src/SprykerEco/Yves/Payone/Form/DataProvider/IdealOnlineTransferDataProvider.php

```diff
@@ -20,10 +21,9 @@ class IdealOnlineTransferDataProvider implements StepEngineFormDataProviderInter
      *
      * @return \Spryker\Shared\Kernel\Transfer\AbstractTransfer
      */
-    public function getData(AbstractTransfer $quoteTransfer)
+    public function getData(AbstractTransfer $quoteTransfer): AbstractTransfer
     {
-        /** @var \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer */
-        if ($quoteTransfer->getPayment() === null) {
+        if ($quoteTransfer instanceof QuoteTransfer && $quoteTransfer->getPayment() === null) {
```

Comment:
Please check like above and other instanceof

Reply:
It is a condition for passing the validation of PHPStan and code-style sniffer.